js/components/portal-dashboard/response-details/response-details.tsx

```diff
@@ -70,6 +80,7 @@ export class ResponseDetails extends React.PureComponent<IProps, IState> {
     const isSequence = activities.size > 1;
 
     const activityId = currentActivity ? currentActivity.get("id") : firstActivity.get("id");
+    const currentActivityWithQuestions = activities.find(activity => activity.get("id") === activityId);
```

Comment:
I'll circle back to this code and see if it can be streamlined.  When we wrote it, I wondered if we could remove some of the code above it.  I'll investigate.

Reply:
I looked at this again.  I think it is fine.